OpenSim/Simulation/Model/Model.h

```diff
@@ -699,6 +710,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
      * Alternatively, set the controls on the model at a given state.
      * Note, this method will invalidate the dynamics of the model,
      * but will mark the controls as valid. (E.g. controllers will not be invoked)
+     *
+     * Note: This method cannot be called during the realization of Dynamics
+     * due to the non-thread-safe controlsCache. If this method must be called
+     * during realizeDynamics, put a lock on the State. (See SimTK::State for
+     * more information)
```

Comment:
What do you think about using doxygen's `\note` or `\warning` here and above? http://www.stack.nl/~dimitri/doxygen/manual/commands.html